libnmstate/schema.py

```diff
@@ -459,3 +461,19 @@ class Ieee8021X:
     PRIVATE_KEY_PASSWORD = "private-key-password"
     CLIENT_CERT = "client-cert"
     CA_CERT = "ca-cert"
+
+
+class Wireguard:
+    TYPE = InterfaceType.WIREGUARD
+    CONFIG_SUBTREE = "wireguard"
+    FWMARK = "fwmark"
+    LISTEN_PORT = "listen-port"
+    PRIVATE_KEY = "private-key"
+    PEERS = "peers"
```

Comment:
The `Wireguard.PEERS` should changed to `Wireguard.Peer.CONFIG_SUBTREE` for consistency.